packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js

```diff
@@ -43,6 +49,38 @@ export default /* @ngInject */ ($stateProvider) => {
         $transition$.params().productId,
       datacenterId: /* @ngInject */ ($transition$) =>
         $transition$.params().datacenterId,
+      datacenterBackups: /* @ngInject */ (
+        $q,
+        datacenterList,
+        dedicatedCloudDatacenterBackupService,
+        productId,
+        datacenterId,
+      ) =>
+        $q.all(
+          map(
+            filter(
+              datacenterList,
+              (datacenter) => datacenter.id !== parseInt(datacenterId, 10),
+            ),
+            (datacenter) =>
+              dedicatedCloudDatacenterBackupService
+                .getBackup(productId, datacenter.id)
+                .then(
+                  (backup) =>
+                    new Backup({
+                      ...backup,
+                      datacenterId: datacenter.id,
+                    }),
+                ),
+          ),
+        ),
+      backupOffersUnderProcess: /* @ngInject */ (datacenterBackups) =>
+        some(
+          datacenterBackups,
+          (backup) => !(backup.isActive() || backup.isInactive()),
```

Comment:
For clarity
```suggestion
          (backup) => !backup.isActive() && !backup.isInactive(),
```
Also, maybe a comment to indicate that there is another state for the backup, as it is disturbing to see:
"If I'm not active and I'm not inactive"

Reply:
Thanks for the feedback @JeremyDec . I have incorporated all your suggestions :)